src/patternfly/components/Tabs/tabs-link.hbs

```diff
@@ -1,6 +1,9 @@
-<{{#if tabs-link--type}}{{tabs-link--type}}{{else}}button{{/if}} class="pf-c-tabs__link{{#if tabs-link--modifier}} {{tabs-link--modifier}}{{/if}}"
+<{{#if tabs-link--IsAnchor}}a{{else}}button{{/if}} class="pf-c-tabs__link{{#if tabs-link--modifier}} {{tabs-link--modifier}}{{/if}}"
```

Comment:
should we update `IsAnchor` anywhere else to be consistent?

Reply:
looks like we use `isLink` and `isAnchorLink`. I used `isLink` in the label PR. I would be in favor of using `isLink` consistently.

```
src//patternfly/components/SimpleList/simple-list-item-link.hbs:<{{#if simple-list-item-link--IsLink}}a{{else}}button{{/if}} class="pf-c-simple-list__item-link{{#if simple-list-item-link--modifier}} {{simple-list-item-link--modifier}}{{/if}}"
src//patternfly/components/SimpleList/simple-list-item-link.hbs:  {{#if simple-list-item-link--IsLink}}
src//patternfly/components/SimpleList/simple-list-item-link.hbs:</{{#if simple-list-item-link--IsLink}}a{{else}}button{{/if}}>
src//patternfly/components/SimpleList/examples/SimpleList.md:{{#> simple-list simple-list-item-link--IsLink="true"}}
marvin:patternfly cmichael$ grep -ri isanchor src/
src//patternfly/components/OverflowMenu/overflow-menu-dropdown-item.hbs:  <{{#if overflow-menu-dropdown-item--IsAnchorLink}}a{{else}}button{{/if}} class="pf-c-dropdown__menu-item{{#if overflow-menu-dropdown-item--modifier}} {{overflow-menu-dropdown-items--modifier}}{{/if}}">
src//patternfly/components/OverflowMenu/overflow-menu-dropdown-item.hbs:  </{{#if overflow-menu-dropdown-item--IsAnchorLink}}a{{else}}button{{/if}}>
```